modules/oe_theme_content_event/templates/oe-theme-content-event-contacts.html.twig

```diff
@@ -11,7 +11,7 @@
     <h2 class="ecl-u-type-heading-2 ecl-u-mt-2xl ecl-u-mt-m-3xl ecl-u-mb-l">{{ 'Contacts'|t }}</h2>
     <div class="ecl-row">
```

Comment:
Since classes `ecl-col-12 ecl-col-m-6` has been removed below, I suppose we don't need `<div class="ecl-row">` also.
Left margin of the contact block is out of the left main border.
<img width="734" alt="Screenshot 2022-03-29 at 14 30 05" src="https://user-images.githubusercontent.com/1183951/160602198-178adcf4-e274-467a-883f-77d0e78a8f0d.png">